modules/installation-and-upgrade/pages/server-setup.adoc

```diff
@@ -75,6 +75,18 @@ The main administration account is the _highest authority account_ within {produ
 For security it is recommended that the main administrator creates _low level admin accounts_ designated for administration of organizations and individual groups.
 ====
 
+[NOTE]
+====
+Newer browser versions can block web access to the {productname} Server FQDN because of HSTS.
```

Comment:
Remember that we decided to disable HSTS per default, so you would run into this problem only in case HSTS was manually enabled from the config file. Not sure if you maybe want to change the wording here, e.g.:

```
Newer browser versions can block web access to the {productname} Server FQDN in case HSTS was enabled.
```

Reply:
ok, thanks for your follow-up!  I created a new PR with additional small modifications for clarity (hopefully):

https://github.com/uyuni-project/uyuni-docs/pull/1601